nfs-ganesha/build/build_rpm

```diff
@@ -41,12 +41,12 @@ cmake -DCMAKE_BUILDER_TYPE=Maintainer -DUSE_FSAL_RGW=YES -DUSE_FSAL_CEPH=YES $WO
 ## Create the source rpm
 echo "Building SRPM"
 rpmbuild \
-    --define "_sourcedir ./dist" \
+    --define "_sourcedir $WORKSPACE/dist" \
     --define "_specdir ." \
```

Comment:
@alimaredia It depends on where your .spec file is, where you want your (src.)rpms to be located, etc. The . is used in place of $PWD in these cases if you were asking about that.